### ISSUE TITLE
[REF] consider-merging-classes-inherited: Skip check if there is `_name` attribute

### ISSUE THREAD

**Issue by moylop260** [OCA/pylint-odoo#92]
Steps:
1. Using the following code:
```python
class MyClass1(model.Models)
   _inherit = 'inherited.model'
   _name = 'my.class'

class MyClass2(model.Models)
   _inherit = 'inherited.model'
   _name = 'other.name'
```

2. Running pylint-odoo

Saw:
 - Emit the message `consider-merging-classes-inherited`

Expected:
 - Don't emit message if there is `_name` attribute because is other table, other model and this way is common for model of reports.

**Comment by moylop260** [comment]
Fixed from https://github.com/OCA/pylint-odoo/pull/51
